src/test/java/org/opensearch/neuralsearch/query/HybridQueryPostFilterIT.java

```diff
@@ -68,7 +68,7 @@ public static void setUpCluster() {
     @SneakyThrows
     public void testPostFilterOnIndexWithSingleShard_whenConcurrentSearchEnabled_thenSuccessful() {
         try {
-            updateClusterSettings("search.concurrent_segment_search.enabled", true);
```

Comment:
Do not review this class as it already merged in main. The change is showing here because we have cherry picked the commit.